lib/grape-swagger.rb

```diff
@@ -231,7 +231,8 @@ def strip_heredoc(string)
             end
 
             def parse_base_path(base_path, request)
-              (base_path.is_a?(Proc) ? base_path.call(request) : base_path) || request.base_path
+              url_scheme = request.env['rack.url_scheme']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.